.travis.yml

```diff
@@ -23,7 +23,7 @@ matrix:
 
 install: composer update $COMPOSER_FLAGS -n
 
-script: vendor/bin/phpunit -v --coverage-clover=coverage.clover
+script: vendor/bin/phpunit --verbose --coverage-clover=coverage.clover
```

Comment:
It's a discussion we had with @webmozart, he prefers to be explicit about this. I don't think that change much, but why not.